dlib/video_io.h

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2021  Davis E. King (davis@dlib.net)
+// License: Boost Software License   See LICENSE.txt for the full license.
+
+#ifdef DLIB_ALL_SOURCE_END
+#include "dlib_basic_cpp_build_tutorial.txt"
+#endif
+
+#ifndef DLIB_VIDEO_IO_ 
+#define DLIB_VIDEO_IO_
+
+#include "video_loader/video_capture.h"
+
+#endif // DLIB_VIDEO_IO_ 
```

Comment:
Ha, you don't have to do all this stuff :)

Really either delete this file and people can just `#include <dlib/video_loader/video_capture.h>` or put the contents of `dlib/video_loader/video_capture.h` in this file and people can `#include <dlib/video_io.h>`.  The latter is more in keeping with overall dlib style.   If in the future the video io stuff gets bigger and warrants a subfolder that can be added and this kind of redirecting header can be added then.